test/files.txt

```diff
@@ -0,0 +1,6 @@
+examples/**/*.ml
+examples/**/*.mli
+!examples/**/*.cppo.*
+!examples/ocamlformat/test/failing/**/*
+!examples/ocamlformat/test/passing/**/*
+!examples/ocaml/testsuite/tests/parse-errors/**/*
```

Comment:
Shouldn't these be in `invalid-files`?

Reply:
I don't think there is a big difference between intentional syntax errors and preprocessors. The result is exactly the same. Why do you think they should be handled differently?

The reason why I added them here, is that some of the `*.cppo.*` and test files could be parsed successfully by accident. If I add the wildcard patterns as `invalid-files`, that would give a warning for every file that is parsed successfully. The only way to fix these warnings, is by listing the files individually. I'm not interested in keeping that list up-to-date.

Conclusion: I'm not interested at all in whether these files can or can't be parsed. So it seems right to exclude them.